cpp/src/arrow/python/builtin_convert.cc

```diff
@@ -16,6 +16,7 @@
 // under the License.
 
 #include <Python.h>
+#include <arrow/util/logging.h>
```

Comment:
can you include this as `"arrow/util/logging.h" with the other arrow includes? https://google.github.io/styleguide/cppguide.html#Names_and_Order_of_Includes